OTGW-firmware.ino

```diff
@@ -60,6 +60,9 @@ void setup() {
   LittleFS.begin();
   readSettings(true);
 
+  CHANGE_INTERVAL_SEC(timerpollsensor, settingGPIOSENSORSinterval, CATCH_UP_MISSED_TICKS); 
```

Comment:
Thanks for catching this.

I think it would be better to group this with the sensor code and move this down to line 87 and insert it before `initSensors();`

Reply:
On second thought; i think this should be moved inside `initSensors()`

Reply:
I agree.  It will be moved to initSensor()

Reply:
I agree. Did a quick fix for myself in my forked version.

Reply:
Actually it should not be in either, as initSensors() is only called on boot. I moved the "change interval" to the read settings routing, that's where logic is stored for the purpose when settings are changed.

Fix is in dev branch now.